templates/default/contentlayer.config.js

```diff
@@ -77,7 +78,9 @@ export default makeSource({
   contentDirExclude: contentLayerExcludeDefaults.concat([
     ".flowershow",
     ".obsidian",
+    ...siteConfig.contentExclude,
   ]),
+  contentDirInclude: [...siteConfig.contentInclude],
```

Comment:
if `siteConfig.contentInclude` is an array, doing `[...siteConfig.contentInclude]` will create a copy of it, which is not necessary.